examples/aws/ecr.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
Would you minder to add more diescription for the notebook? so that user can know what they can get from the notebook. Thanks!

Reply:
Yeah. I add comments on this notebook.